deltacat/benchmarking/conftest.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import io
+
+import boto3
+import pyarrow as pa
+import pyarrow.fs as pafs
+import pyarrow.parquet as papq
+import pytest
+
+from deltacat.utils.pyarrow import s3_file_to_table
+from deltacat.types.media import (
+    ContentEncoding,
+    ContentType,
+)
+
+
+def pyarrow_read(path: str, columns: list[str] | None = None) -> pa.Table:
+    assert path.startswith("s3://")
```

Comment:
Minor (same for other asserts):
```suggestion
    assert path.startswith("s3://"), f"Expected file path to start with 's3://', but got {path}."
```